src/authentication/DuplicateFlowModal.tsx

```diff
@@ -46,12 +46,12 @@ export const DuplicateFlowModal = ({
     try {
       await adminClient.authenticationManagement.copyFlow({
         flow: name,
-        newName: form.name,
+        newName: form.alias,
       });
       if (form.description !== description) {
         const newFlow = (
           await adminClient.authenticationManagement.getFlows()
-        ).find((flow) => flow.alias === form.name)!;
+        ).find((flow) => flow.alias === form.alias)!;
```

Comment:
We should not be using the non-null assertion operator here as the result from a `find()` operation could possibly be `undefined`.
```suggestion
        ).find((flow) => flow.alias === form.alias);
```

Reply:
how can it not be found as it's just been created 1 line above

Reply:
Of course, but there are no guarantees.